### PR TITLE
Contrast Ratio Demo

### PR DESCRIPTION
**Pull request by kitkat119**
- adds demo where user can select colors for text/background and see the contrast ratio/WCAG rating
- also changes usecase buttons to a dropdown

### CODE REVIEW

**Comment by gvonkoss** [comment]
Made a few changes based on Kats 👌 work, looks like this now:

contrast

File: demos/src/contrast-checker.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	const backgroundHex = getComputedStyle(docElem).getPropertyValue(`--o-colors-${backgroundColor}`);
+
+	const ratingMessage = document.querySelector('.rating-message');
+	const ratioValue = document.querySelector('.contrast-ratio');
+	const wcagRating = document.querySelector('.wcag-rating');
+	const ratio = contrastRatio.oColorsGetContrastRatio(textHex, backgroundHex);
+	const rating = contrastRatio.oColorsGetWCAGRating(ratio, textColor, backgroundColor);
+
+	ratingMessage.className = `rating-message rating-result--${rating.wcagRating.toLowerCase()}`;
+	ratingMessage.innerHTML = rating.message;
```
[the remark this answers is not in the document]
Comment (gvonkoss) [reply demos/src/contrast-checker.js]: 👌

File: demos/src/contrast-checker.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+	ratioValue.innerHTML = `Contrast ratio: ${ratio}`;
```
Comment (JakeChampion) [inline demos/src/contrast-checker.js]: Same comment as above, we should use `textContent` -- https://github.com/Financial-Times/o-colors/pull/154/files#r188284675

File: demos/src/contrast-checker.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+	wcagRating.className = `wcag-rating rating-result--${rating.wcagRating.toLowerCase()}`;
+	wcagRating.innerHTML = `WCAG ${rating.wcagRating}`;
```
Comment (JakeChampion) [inline demos/src/contrast-checker.js]: Same comment as above, we should use `textContent` -- https://github.com/Financial-Times/o-colors/pull/154/files#r188284675

**Pull request merged**